app/javascript/components/CaseNoteForm/index.js

```diff
@@ -284,8 +295,11 @@ CaseNoteForm.propTypes = {
   description: PropTypes.string,
   internal: PropTypes.bool,
   display: PropTypes.string,
-  id: PropTypes.number,
+  caseNoteId: PropTypes.number,
   participantId: PropTypes.number.isRequired,
+  incrementNumCaseNotes: PropTypes.func,
+  appendCaseNote: PropTypes.func,
+  updateCaseNote: PropTypes.func,
 };
 
 CaseNoteForm.defaultProps = {
```

Comment:
Just a general comment, there are a lot of functions where you use arrow notation and some more that you bind "this". Choose one and stick with it :)

Reply:
Is there a preferred one/ one that's considered "better"? Rule I've been following is that if it's a class component I'll do function definitions with bind "this" (or if the existing component already has functions that are bound) and if it's a functional component I'll do arrow notation.

Reply:
I've heard that convention as well.